README.md

```diff
@@ -9,8 +8,52 @@ You can also ask BotKube to execute kubectl commands on k8s cluster which helps
 
 ![](botkube-title.jpg)
 
-## Getting started
-Please follow [this](https://www.botkube.io/installation/) for complete BotKube installation guide.
+## Installation
+
+### Stable release (Recommended)
+
+Please follow [this guide](https://www.botkube.io/installation/) to install latest stable version of BotKube.
+
+### From develop branch
```

Comment:
We should separate out this section may be to contributors doc. User might end up copying following steps when they actually want to use stable branch.